src/main/java/com/bobocode/svydovets/beans/definition/BeanDefinition.java

```diff
@@ -25,4 +25,4 @@ public Class<?> getBeanClass() {
     public void setBeanClass(Class<?> beanClass) {
         this.beanClass = beanClass;
     }
-}
+}
```

Comment:
Can you please add an empty line at the end of the file to stop GitHub from complaining.